.github/workflows/build-test-deploy.yml

```diff
@@ -398,6 +437,36 @@ jobs:
       - run: |
           cachix watch-exec -j 16 -c 0 composable-community nix -- build .#check-${{ matrix.runtime }}-integration-tests --no-update-lock-file --show-trace -L       
 
+  package-simnode-tests:
```

Comment:
just comment, so we can follow terminology of nix, nixops. build, check, deploy.  so package is build.

Reply:
not for now. just note.

Reply:
it's a binary